test/TestsHelper.js

```diff
@@ -478,6 +479,13 @@ TestsHelper.execCmdWithAssertion = function (cliCmd, cmdOptions, apiOptions, sna
         // ensure line separators are always the same
         finalOutput = finalOutput.replace(/\r\n/g, '\n');
 
+        // regex expression or text to remove from the output
+        if (textToRemove) {
+          const matchedText = finalOutput.match(textToRemove)[0];
+          expect(matchedText).to.exist;
+          finalOutput = finalOutput.replace(matchedText, '');
+        }
```

Comment:
I can see why you did this but at the same time I feel that the function gets even more complicated. It already accepts too many arguments and is confusing enough. Currently, I can't think of a case where we wouldn't want to replace the date, any date, not just for `flex status`.